core/sessions/orm.go

```diff
@@ -52,58 +56,90 @@ func NewORM(db *sqlx.DB, sessionDuration time.Duration, lggr logger.Logger) ORM
 	return &orm{db, sessionDuration, lggr.Named("SessionsORM")}
 }
 
-// FindUser will return the one API user, or an error.
-func (o *orm) FindUser() (User, error) {
-	return o.findUser()
+// FindUser will attempt to return an API user by email.
+func (o *orm) FindUser(email string) (User, error) {
+	return o.findUser(email)
 }
 
-func (o *orm) findUser() (user User, err error) {
-	sql := "SELECT * FROM users ORDER BY created_at desc LIMIT 1"
-	err = o.db.Get(&user, sql)
+// FindUserByAPIToken will attempt to return an API user by email.
+func (o *orm) FindUserByAPIToken(apiToken string) (user User, err error) {
+	sql := "SELECT * FROM users WHERE token_key = $1"
+	err = o.db.Get(&user, sql, apiToken)
 	return
 }
 
-// AuthorizedUserWithSession will return the one API user if the Session ID exists
-// and hasn't expired, and update session's LastUsed field.
+func (o *orm) findUser(email string) (user User, err error) {
+	sql := "SELECT * FROM users WHERE email = $1"
+	err = o.db.Get(&user, sql, email)
+	return
+}
+
+// ListUsers will load and return all user rows from the db.
+func (o *orm) ListUsers() (users []User, err error) {
+	sql := "SELECT * FROM users ORDER BY role ASC;"
```

Comment:
May I suggest tie-breaking on email?

Reply:
Ah yeah that'd be more intuitive. Updated